src/cluster.bootstrap/Akka.Management.Cluster.Bootstrap/ContactPoint/HttpBootstrapJsonProtocol.cs

```diff
@@ -41,14 +41,14 @@ public ClusterMember(Address node, long nodeUid, MemberStatus status, ImmutableH
         
         public sealed class SeedNodes
         {
-            public SeedNodes(Address selfNode, ImmutableHashSet<ClusterMember> nodes)
+            public SeedNodes(Address selfNode, ImmutableList<ClusterMember> nodes)
```

Comment:
Is it possible that this could allow duplicates?

Reply:
Nope, it can't - comes from the `Cluster` object itself which already de-dupes them.